src/stores/WidgetsStore.ts

```diff
@@ -44,7 +45,8 @@ import {
     CatalogPlotWidgetStoreProps,
     ACTIVE_FILE_ID,
     CatalogPlotType,
-    LayerListWidgetStore
+    LayerListWidgetStore,
+    PVGeneratorWidgetStore
```

Comment:
inconsistent capitalisation of `PV`. I think best to use `PvGenerator`

Reply:
done. has replaced all capitalisation PV into PvGenerator.